pkg/kubelet/types/labels_test.go

```diff
@@ -0,0 +1,129 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package types
+
+import (
+	"testing"
+)
+
+func TestGetContainerName(t *testing.T) {
+	var cases = []struct {
+		labels        map[string]string
+		containerName string
+	}{
+		{
+			labels: map[string]string{
+				"io.kubernetes.container.name": "c1",
+			},
+			containerName: "c1",
+		},
+		{
+			labels: map[string]string{
+				"io.kubernetes.container.name": "c2",
+			},
+			containerName: "c2",
+		},
+	}
+	for _, data := range cases {
+		containerName := GetContainerName(data.labels)
+		if containerName != data.containerName {
```

Comment:
nit, for all the equality checks using testify's "assert.Equal" is more readable